deploy/infra/noazurejson/README.md

```diff
@@ -0,0 +1,53 @@
+# Deploy AAD Pod Identity with a dedicated service principal
+
+## The why
+
+Goal: To enable user to use a separate service pricipal (aad-pod-identity admin service principal) other than the cluster service princial and to move away from `/etc/kubernetes/azure.json`.
+
+Users now have the option to deploy aad-pod-identity with a separate service principal which is together with its secret and other configurations stored in a Kubernetes secret object.
+
+## Permissions
+
+The permission of the admin service principal needs to be 'Contributor' role over the scope of node resource group starting with "MC_".
+
+Create a new service principal with the permission:
+
+```
+az ad sp create-for-rbac -n "<sp_name>" --role "Contributor" --scopes "/subscriptions/<subscription-id>/resourceGroups/<MC_node_resource_group>" 
+```
+
+> Note the `appId` (client id), `password` (secret) and `tenant` from the resulting json, which will be used in creating the admin secret.
+
+Or assign the permission for an existing service principal:
+
+```
+az role assignment create --role "Contributor" --assignee <sp_id> --scope "/subscriptions/<subscription-id>/resourceGroups/<MC_node_resource_group>"
+```
+
+For any subsequent user assigned managed identity that's intended for a pod, it's also required to grant the service principal 'Managed Identity Operator' permission (also stated [here](../../../README.md#6-set-permissions-for-mic)):
+
+```
+az role assignment create --role "Managed Identity Operator" --assignee <sp_id> --scope <resource id of the managed identity>
+```
+
+## Create the admin secret
+
+The `aadpodidentity-admin-secret` contains the following fields:
+
+- Cloud: `<base64-encoded-cloud>`
+- SubscriptionID: `<base64-encoded-subscription-id>`
+- ResourceGroup: `<base64-encoded-resource-group>`
+- VMType: `<base64-encoded-vm-type>`
+- TenantID: `<base64-encoded-tenant-id>`
+- ClientID: `<base64-encoded-client-id>`
+- ClientSecret: `<base64-encoded-client-secret>`
+
+> Use `echo -n 'secret-content' | base64` to create a base64 encoded string.
+
+'Cloud' should be chosen from the following case-insensitive values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `AzureGermanCloud` (values taken from [here](https://raw.githubusercontent.com/Azure/go-autorest/master/autorest/azure/environments.go)). 'ResourceGroup' is the node resource group where the actual virtual machines or virtual machine scale set resides. 'VMType' is optional and can be one of these values: `standard` for normal virtual machine nodes, and `vmss` for cluster deployed with a virtual machine scale set. 'TenantID', 'ClientID' and 'ClientSecret' are service principal's `tenant`, `appId`, `password` respectively.
```

Comment:
Can we organize these params and possible values as a list here instead of one paragraph?

Reply:
put possible values/explanations as sub-list under each param